resources/aws/securitygroup.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/clustertpr"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+const (
+	sshPort = 22
+)
+
+type SecurityGroup struct {
+	Description string
+	GroupName   string
+	VpcID       string
+	Cluster     clustertpr.Cluster
```

Comment:
I'm not so sure this type needs to have access to the whole TPR. But I understand that this could be a compromise to get us running. In that case though, we should address this later and have an issue to track it.